package.json

```diff
@@ -42,10 +42,10 @@
   "main": "index.js",
   "types": "index.d.ts",
   "dependencies": {
-    "@types/bluebird": "3.5.4",
-    "@types/node": "6.0.41",
-    "@types/reflect-metadata": "0.0.4",
-    "@types/sequelize": "4.0.73",
+    "@types/bluebird": "^3.5.15",
```

Comment:
Let us keep the exact versions here. Here is a discussion about it: https://github.com/RobinBuschmann/sequelize-typescript/issues/142